packages/utils/recursive-parser.test.js

```diff
@@ -25,3 +25,55 @@ defineTest(
 	},
 	"init"
 );
+
```

Comment:
How does it behave if you tried to add property that already exists? Could you define a test for this case please?

Reply:
^ Otherwise this would just be the `init` feature

Reply:
`entry` was already present in the input fixture.  I've added `mode` as well to make it clearer.